quicktests/overlaying/tests/realistic/stocks.js

```diff
@@ -12,13 +12,13 @@ function run(svg, data, Plottable) {
   }
 
   // load GOOG
-  d3.csv("/quicktests/data/GOOG_20140401_20140901.csv")
+  d3.csv("/quicktests/overlaying/data/GOOG_20140401_20140901.csv")
     .get(function(error, rows) {
       var goog = rows.reverse();
       goog.forEach(processDatum);
 
       // load AAPL
```

Comment:
The HTTP server we run will root to whichever folder we run grunt from. Hence, / is not the linux `/` but the server `/`